readme.adoc

```diff
@@ -639,69 +635,68 @@ See `link:examples/jfx.java[]` and `link:examples/jfxtiles.java[]` for examples
 They expect their plugins to be named like `<cmd>-<plugin>`, i.e. `kubectl-myplugin`.
 
-Furthermore some of them, particularly `kubectl` currently require the file to start with `#!` otherwise you get a `excc format error`.
+Furthermore some of them, particularly `kubectl` currently require the file to start with `#!` otherwise you get a `exec format error`.
 
 `jbang` has a bit of auto-magic to help in both cases.
 
-=== kebab casing of filename
+=== Kebab casing of filename
 
 `jbang` lets you name your file without a `.java` or `.jsh` extension, such
 as `kubectl-myplugin`. `jbang` will in this case copy the file to a temporary
 directory using kebab-case to map the name to a proper java class name.
 
-i.e. If you make a file called `kubectl-myplugin` then `jbang` will assume the actual class name to launch
-will be `KubectlMyPlugin`.
+For example, if you make a file called `kubectl-myplugin` then `jbang` will assume the actual class name to launch
+to be `KubectlMyplugin`.
 
 Note, similar is done when using `jbang edit`, here the symbolic link will be made so the IDE will treat it as
 regular camel cased java class.
 
-Note: If you do not follow this naming pattern you will get a compile error as `javac` expect the public class name are equal to the filename.
+NOTE: If you do not follow this naming pattern you will get a compile error as `javac` expects both the public class and file names to be equal.
 
 
 === she-bang auto-stripped
 
-For extension less scripts, you can put `#!' header in beginning to let apps recognize
-it is to be treated as a script. To avoid issues when compiling `jbang` will remove
-that line before compiling.
+For extension less scripts, you can put `#!' header at the beginning to let apps recognize
+it is to be treated as a script. To avoid issues when compiling, `jbang` will remove
+that line before compilation.
 
 For now this is required for `kubectl` plugin but not `git`. https://github.com/kubernetes/kubectl/issues/822[Issue opened] on this limitation.
 
 == Java version
 
-jbang will by default first use `JAVA_HOME` and if not avialble the `PATH` to locate the `java` executable to run the script with.
+`jbang` will by default use `JAVA_HOME` and if not available, check the the `PATH` to locate the `java` executable to run the script with.
 
 If your script requires a specific or minimal version of Java you can use `//JAVA <version>(+)`.
 
-If jbang finds a java executable using `JAVA_HOME` or `'PATH` which satisfies the stated java version jbang will use it.
+If jbang finds a java executable using `JAVA_HOME` or `PATH` which satisfies the stated java version jbang will use it.
 If no such version is found it will automatically download and install it into jbang's cache (`~/.jbang/cache/jdks` by default).
 
 Examples:
 
-`//JAVA 11` will force use of Java 11.
-`//JAVA 13+` will require at least java 13. Java 13 or higher will be sued.
+  //JAVA 11  will force use of Java 11.
```

Comment:
do you really mean to have the "will force use of java 11" in code style rather than text ?

Reply:
You are right, I will fix it.
What I saw is that it was written in a single line. Is still correct to asume it the sentences should be in 2 lines?

Reply:
yes, two lines are fine.